.appveyor.yml

```diff
@@ -5,7 +5,6 @@ clone_folder: C:\projects\php-cs-fixer
 
 environment:
     matrix:
-        - php_ver: 7.1.2
```

Comment:
What was wrong with this version?

Reply:
it was freezing on phpunit execution

Reply:
ref https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3188